highfidelity/hifi
# Don't send avatars to previous lookup on server restart

## Test Plan:
- Point a place name to a domain (ie. Create the domain "PlaceNameA" and point "PlaceNameB" to `hifi://PlaceNameA/X,Y,Z)
- Go to "PlaceNameB"
- Move your avatar
- Restart domain

On the current release, you'll notice that upon reconnection, the avatar will be taken back to "PlaceNameB".
On this PR, the avatar should stay where it is.
